Ports/.port_include.sh

```diff
@@ -912,14 +924,14 @@ parse_arguments() {
                 do_installdepends
             fi
             if [ -d "$workdir" ] && [ ! -d "$workdir/.git" ]; then
-                if prompt_yes_no "- Would you like to clean the working direcory (i.e. ./package.sh clean)?"; then
+                if prompt_yes_no "- Would you like to clean the working directory (i.e. ./package.sh clean)?"; then
                     do_clean
                 fi
             fi
             do_dev
             ;;
         *)
-            >&2 echo "I don't understand $1! Supported arguments: fetch, patch, configure, build, install, installdepends, interactive, clean, clean_dist, clean_all, uninstall, showproperty, generate_patch_readme, dev."
+            >&2 echo "I don't understand $1! Supported arguments: fetch, patch, configure, build, install, installdepends, interactive, clean, clean_dist, clean_all, uninstall, showproperty, generate_patch_readme, generate_available_ports, dev."
```

Comment:
This new command should be added to `Ports/README.md` as well

Reply:
it is mentioned in “dev” section. thereʼs a bunch of other commands not described there, e.g. `generate_patch_readme`…

i guess that youʼd like to have them also there? 🤔
i would give a sentence for each from myself, but i put whatever you provide me with (helping hand highly appreciated!). 🙂

Reply:
Hm, I guess mentioning it under `dev` is enough!